roles/test-idp/files/saml20-idp-remote.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * SAML 2.0 remote IdP metadata for SimpleSAMLphp.
+ *
+ * Remember to remove the IdPs you don't use from this file.
+ *
+ * See: https://simplesamlphp.org/docs/stable/simplesamlphp-reference-idp-remote
+ */
+
+$metadata['https://test-idp.sram.surf.nl/saml/saml2/idp/metadata.php'] = [
```

Comment:
hmm, en waarom is deze remote metadata eigenlijk nodig?  Het is toch een pure IdP?

Reply:
Dit is de interne kortsluiting tussen de ssPHP SP en IdP, om zichzelf te kunnen testen. ssPHP kent ook een auth test, maar dan zie je alleen de interne attribuutnamen, niet de name2oid en name2urn translatie, die komt pas later in het proces, richting de SP.